numpy/core/arrayprint.py

```diff
@@ -200,6 +200,8 @@ def set_printoptions(precision=None, threshold=None, edgeitems=None,
     -----
     `formatter` is always reset with a call to `set_printoptions`.
 
+    `printoptions` context manager can be used to set the values temporarily.
+
```

Comment:
This just cries out for an example, it seems like a natural continuation of "To put back the default options ...". I think "Use `printoptions` as a context manager to ..." is clearer, no?

Reply:
> I think "Use printoptions as a context manager to ..." is clearer, no?

Probably yes, I agree that it is better.
Do we need to move this to `See Also` from `Notes`?

Reply:
I would prefer it be in the `Examples` section with some example code.

Reply:
Yes, but some examples already exist in `printoptions` documentation. It seems to me the reference and slight clarification in `set_printoptions` will be enough.

Reply:
none of the examples use `printoptions` as a context manager.

Reply:
huh, here I see examples with `printoptions` context manager.
https://numpy.org/devdocs/reference/generated/numpy.printoptions.html#numpy.printoptions

Perhaps, these examples are not detailed enough?

Reply:
Could you copy one to the docstring of `set_printoptions`, even in place of the "To put back the default options" one? Then the text would become "To temporarily override options, use `printoptions` as a context manager" which is much nicer than the example that exists there

Reply:
Got it. It looks good to me.